starter-code/routes/index.js

```diff
@@ -1,9 +1,52 @@
-var express = require('express');
```

Comment:
this file should be for the main page route "/", then you can create something like auth routes file to put all auth stuff in there